tests/test_coils.py

```diff
@@ -1505,18 +1575,21 @@ def test_initialize_helical():
 
 
 @pytest.mark.unit
-def test_FourierPlanarCoil_from_values_orientation():
-    """Test that fitting with FourierPlanarCoil preserves orientation."""
+def test_planar_coil_from_values_orientation():
+    """Test that fitting to a planar coils preserves orientation."""
     # easiest to check this is working using a coil, as can use
     # biot savart to see if B is same or not
     # tests fix for GH #1715
     coil_XYZ = FourierXYZCoil(
         current=1e6, X_n=[0, 10, 2], Y_n=[-2, 0, 0], Z_n=[0, 0, 0.1]
     )
-    coil_planar = coil_XYZ.to_FourierPlanar(N=1)
+    coil_planar1 = coil_XYZ.to_FourierPlanar(N=1)
+    coil_planar2 = coil_XYZ.to_FourierXY(N=1)
```

Comment:
Since `FourierXY` uses an arbitrary angle, idt this is actually an issue. But am fine with keeping the test